charts/trusted-artifact-signer/templates/consoleclidownload-cosign.yaml

```diff
@@ -8,6 +8,10 @@ spec:
     manage sigstore artifacts.
   displayName: cosign - Command Line Interface (CLI)
   links:
-  - href: "https://{{ $.Values.configs.clientserver.name }}-{{ $.Values.configs.clientserver.namespace }}.{{ $.Values.global.appsSubdomain }}/clients/cosign"
+  - href: "https://{{ $.Values.configs.clientserver.name }}-{{ $.Values.configs.clientserver.namespace }}.{{ $.Values.global.appsSubdomain }}/clients/linux/cosign-linux-amd64"
```

Comment:
please change to `clients/linux/cosign` (also remove the tag in Dockerfile, as suggested below) and do the same for the other consoleclidownoads.

Reply:
Ill get on it right away!